2696. Minimum String Length After Removing Substrings/solution.py

```diff
@@ -9,3 +9,5 @@
 		# in both loops, if a match is found, cut those two characters out of the string and reform as a new string
 			# if this happens decrement strlen by 2
   # return strlen after the loops have finished
+
+# trying to make a PR
```

Comment:
A general comment - I think because you are not yet familiar with the syntax of python, your psuedocode talk more about what you need to do in code versus why you're doing it. That's totally fine for now! As you get more familiar with Python, you can switch the comments like "declare a variable str len..." or "loop through the string while incrementing by 1" more to comments like, "iterate through s to search for matches for AB or CD" w/o going into as much details that will be very clear from the code.

Reply:
Last comment, generally comment and code lines shouldn't go past 80 characters. Leetcode interface doesn't make that easy but just end your lines manually at "page width" which is like the width of git hub's code view area. It doesn't need to be exact but it'll make it easier to review.